frontend/src/App.css

```diff
@@ -0,0 +1,216 @@
+.app{
+    min-height: 100vh;
+}
+/* Navbar */
+.m-b-0{
+    margin-bottom: 0rem;
+}
+.nav-list-none{
+    display: none;
+}
+
+a{
+    color: #fff;
+}
+.transform-lowercase{
+   text-transform: lowercase;
+   font-family: cursive;
+}
+.badge-nav-cart{
+    position: absolute;
+    color: white;
+    top: -10px;
+    left: 42px;
+}
+.badge-nav-wishlist{
+    position: absolute;
+    color: white;
+    top: -10px;
+    left: 65px;
+}
+.navbar-fixed-top{
+    position: fixed;
+    z-index: 1;
+    width: 100%;
+    top: 0;
+}
+/* Landing */
+.landing{
+    background: url("./assets/images/landing.jpg") no-repeat center center/cover;;
+    position: relative;
+    height: 100vh;
+    width: 100%;
+}
+.landing-inner-center{
+    display: flex;
+    align-items: center;
+    height: 100%;
+    justify-content: flex-end; 
+}
+.lead{
+    color: var(--white);
+    font-size: 1.5rem;
+    margin: 0.5rem;
+}
+
+
+.d-flex{
```

Comment:
what is the significance of d- here?

Reply:
changed this to display-flex more descriptive name